front/components/Layout/Header/RightNav/UserCard/UserCard.spec.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { MockedProvider } from '@apollo/client/testing';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import Home from './index';
+
+// eslint-disable-next-line
+const useRouter = jest.spyOn(require('next/router'), 'useRouter');
+// eslint-disable-next-line
+const useViewer = jest.spyOn(require('hooks/auth'), 'useViewer');
+
+test('UserCard expandLess', () => {
+  const mocks = [];
+
+  useRouter.mockImplementation(() => ({
+    route: '',
+    pathname: '',
+    query: '',
+    asPath: '',
+  }));
+
+  useViewer.mockImplementation(() => ({
+    username: 'toto',
+    email: 'tata',
+  }));
+
+  render(
+    <MockedProvider mocks={mocks} addTypename={false}>
+      <Home />
+    </MockedProvider>,
+  );
```

Comment:
We probably want to have a custom `render` function that provides a `Wrapper` by default with all of our global providers (like apollo).

See [here](https://github.com/kentcdodds/bookshelf/blob/main/src/test/app-test-utils.js) for an example